### DISCUSSION TITLE
Awesome Plug-in/Question about Quarterly Dates

### DISCUSSION THREAD

**Discussion by edwarddarrah** [YU000jp/logseq-plugin-show-weekday-and-week-number#155] [General]
Thanks for all your hard work on Logseq and your plug-ins. I had wanted Logseq to expand the journal notes for a while and was blown away when I came across your plugin-show-weekday-and-week-number. I am still getting a feel for the settings. One thing I'm having trouble with getting quarterly dates to work; it is are toggled on in settings. I would like assistance with this. 

Do you have plans to integrate into the Database version?

Thanks

**Comment by YU000jp** [comment]
Sorry, but this will only be of limited use until we move to the db version. Regarding your question about support for the db version, this plugin will most likely not be able to handle it. This is because it is very likely that the “/” will no longer be used as a hierarchical function. To access any quarter page, please access it from the hierarchy. This plugin is not available in the db version and must be redesigned.
